pkg/handlers/v1/nexposeassetproducer.go

```diff
@@ -64,8 +65,10 @@ func (h *NexposeScannedAssetProducer) Handle(ctx context.Context, in ScanInfo) {
 				errChan = nil
 			} else {
 				switch err.(type) {
-				case *assetfetcher.NeverBeenScanned:
-					stater.Count("assetskipped", 1, fmt.Sprintf("site:%s", in.SiteID), fmt.Sprintf("reason:%s", "neverbeenscanned"))
+				case *assetfetcher.ScanIDForLastScanNotInAssetHistory:
```

Comment:
If this gets pushed, then we will need to change some terraform/sfx stuff.
Task: Will you remind me to do this later?

Reply:
PSD-300

Reply:
actually i don't think you would need to change the terraform stuff, because we group/sum by reason.